frontend/components/ManageStakingPage/StakingContractSection/MigrateButton.tsx

```diff
@@ -132,28 +132,19 @@ export const MigrateButton = ({
               // update service
               await ServicesService.updateService({
                 serviceConfigId,
-                partialService: {
-                  chain_configs: {
-                    ...selectedService.chain_configs,
-                    // overwrite defaults with chain-specific configurations
-                    ...Object.entries(selectedService.chain_configs).reduce(
-                      (acc, [middlewareChain, config]) => {
+                partialServiceTemplate: {
+                  configurations: {
+                    ...Object.entries(serviceTemplate.configurations).reduce(
```

Comment:
not sure this will work for e.g. Optimus, doesn't seem correct to pass `staking_program_id` to each chain that can be in config =\

Reply:
we only stake on home chain afaik, more just a redundancy issue here i think 🤔